content/docs/introduction/overview.md

```diff
@@ -12,26 +12,26 @@ monitoring and alerting toolkit originally built at
 [SoundCloud](http://soundcloud.com). Since its inception in 2012, many
 companies and organizations have adopted Prometheus, and the project has a very
 active developer and user [community](/community). It is now a standalone open source project
-and maintained independently of any company. To emphasize this and clarify
+and maintained independently of any company. To emphasize this, and to clarify
 the project's governance structure, Prometheus joined the
 [Cloud Native Computing Foundation](https://cncf.io/) in 2016
-as the second hosted project after [Kubernetes](http://kubernetes.io/).
+as the second hosted project, after [Kubernetes](http://kubernetes.io/).
 
-For a more elaborate overview, see the resources linked from the
+For more elaborate overviews of Prometheus, see the resources linked from the
 [media](/docs/introduction/media/) section.
 
 ### Features
 
 Prometheus's main features are:
 
-* a multi-dimensional [data model](/docs/concepts/data_model/) (time series identified by metric name and key/value pairs)
+* a multi-dimensional [data model](/docs/concepts/data_model/) with time series data identified by metric name and key/value pairs
 * a [flexible query language](/docs/querying/basics/)
   to leverage this dimensionality
 * no reliance on distributed storage; single server nodes are autonomous
 * time series collection happens via a pull model over HTTP
 * [pushing time series](/docs/instrumenting/pushing/) is supported via an intermediary gateway
 * targets are discovered via service discovery or static configuration
-* multiple modes of graphing and dashboarding support
+* multiple modes of graphing and dashboard support
```

Comment:
This is fine as is, one with a gerund the other other without is odd.

Reply:
Dashboarding isn't a word was my objection. :)

Reply:
There's more hits for "dashboarding" than "verbing" :)

Reply:
LOL! Okay :)